pkg/etcdpasswd/cmd/get.go

```diff
@@ -0,0 +1,55 @@
+package cmd
+
+import (
+	"context"
+	"errors"
+	"fmt"
+
+	"github.com/spf13/cobra"
+)
+
+var getCmd = &cobra.Command{
+	Use:   "get CONFIG",
+	Short: "get configurations",
+	Long: `Usage: etcdpasswd get CONFIG
```

Comment:
This line looks spurious.

Reply:
I could not understand what spurious means here for.
Tried to use another expression, but please notify me if it is wrong.

Reply:
The total line should be removed as cobra will display this.

Reply:
Is it better to remove the line entirely?
I changed the expression and now the help looks as follows.

```
$ ./etcdpasswd get --help
get configurations.

CONFIG is one of:
    start-uid:      the beginning UID assigned to managed users.
    start-gid:      the beginning GID assigned to managed users.
    default-group:  default primary group.
    default-groups: comma-separated list of supplementary groups.
    default-shell:  default shell program.

Usage:
  etcdpasswd get CONFIG [flags]

Flags:
  -h, --help   help for get

Global Flags:
      --config string      configuration file path (default "/etc/etcdpasswd/config.yml")
      --logfile string     Log filename
      --logformat string   Log format [plain,logfmt,json]
      --loglevel string    Log level [critical,error,warning,info,debug]
```